src/fiatExchanges/FiatExchangeAmount.tsx

```diff
@@ -84,6 +85,7 @@ function FiatExchangeAmount({ route }: Props) {
   const localCurrencyCode = useLocalCurrencyCode()
   const dailyLimitCusd = useSelector(cUsdDailyLimitSelector)
   const exchangeRates = useSelector(localCurrencyExchangeRatesSelector)
+  const mostRecentFiatAccount = useSelector(mostRecentFiatAccountSelector)!
```

Comment:
i'm confused. doesn't the trailing ! mean you're asserting it's non-null? what if they haven't created a fiat account yet?

Reply:
also, I didn't realize this before but the trailing `!` doesn't result in any type checks taking place, it just turns off compile-time errors for nullable values. at runtime it could still be null and pass right by this line. terrifying